multixlat test does not link properly on Debian sid and Ubuntu jammy

```
/bin/bash ../libtool  --tag=CC   --mode=link gcc  -g -O2 -ffile-prefix-map=/build/libkdumpfile-0.4.1+20220811gitfa8f755=. -fstack-protector-strong -Wformat -Werror=format-security  -Wl,-z,relro -Wl,-z,now -o multixlat multixlat.o ../src/kdumpfile/libkdumpfile.la 
libtool: link: gcc -g -O2 -ffile-prefix-map=/build/libkdumpfile-0.4.1+20220811gitfa8f755=. -fstack-protector-strong -Wformat -Werror=format-security -Wl,-z -Wl,relro -Wl,-z -Wl,now -o .libs/multixlat multixlat.o  ../src/kdumpfile/.libs/libkdumpfile.so
/usr/bin/ld: multixlat.o: undefined reference to symbol 'addrxlat_addrspace_name@@LIBADDRXLAT_0'
/usr/bin/ld: /build/libkdumpfile-0.4.1+20220811gitfa8f755/src/addrxlat/.libs/libaddrxlat.so.3: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
make[2]: *** [Makefile:1301: multixlat] Error 1
```
[libkdumpfile_0.4.1+20220811gitfa8f755-1_i386.build.gz](https://github.com/ptesarik/libkdumpfile/files/9312089/libkdumpfile_0.4.1%2B20220811gitfa8f755-1_i386.build.gz)

--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -84,6 +84,7 @@ dumpdata_LDADD = \
 multiread_LDADD = \
 	$(top_builddir)/src/kdumpfile/libkdumpfile.la
 multixlat_LDADD = \
+	$(top_builddir)/src/addrxlat/libaddrxlat.la \
 	$(top_builddir)/src/kdumpfile/libkdumpfile.la
 nometh_LDADD = \
 	$(top_builddir)/src/addrxlat/libaddrxlat.la

